datapirates/challenge.md

```diff
@@ -0,0 +1,26 @@
+# Data Pirates challenge
+
+Welcome to Data Pirates challenge.
+
+
+## Scenario:
+
+The Internet Movie Database (IMDb) is an online database of information about music, movies, movies, TV shows and games and television games computer. [Wikipedia]. This challenge consist to collect information from this site and deliver it other format.
+
+
+## Requirements:
+
+ * Get 500 titles from each genre
+ * Sorted by 'Rating'
+ * This app must run with only one command
+ * The output format is JSONL
+ * Write each genre in one output file
+
+
+Delivery:
+
+The code should be sent through github, with a usage documentation or something.
```

Comment:
Suggestion: "...or something that helps to get your app running."

Reply:
How about `with documentation how to run it`. you don't want to guess how to run the code from someone else, right? :)

Reply:
indeed

thanks for your suggestions :)